<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="InvalidPipelineStoreException.xml" source-language="en-US" target-language="it-IT">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac59bbe4e4cfa38c1fd53c0b87c4aacce4d1e815312.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">9bbe4e4cfa38c1fd53c0b87c4aacce4d1e815312</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.AddIn.Hosting.InvalidPipelineStoreException">
          <source>The exception that is thrown when a directory is not found and the user does not have permission to access the pipeline root path or an add-in path.</source>
          <target state="translated">Eccezione generata quando la ricerca di una directory ha esito negativo e l'utente non è autorizzato ad accedere al percorso radice della pipeline o a un percorso del componente aggiuntivo.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AddIn.Hosting.InvalidPipelineStoreException">
          <source>Unlike <ph id="ph1">&lt;xref:System.IO.DirectoryNotFoundException&gt;</ph>, this exception does not provide any path information.</source>
          <target state="translated">A differenza dell'eccezione <ph id="ph1">&lt;xref:System.IO.DirectoryNotFoundException&gt;</ph>, questa eccezione non fornisce alcuna informazione sul percorso.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AddIn.Hosting.InvalidPipelineStoreException">
          <source>This prevents a malicious user from deliberately specifying erroneous paths and using the information returned by the exception to determine actual paths.</source>
          <target state="translated">Ciò impedisce a un utente malintenzionato di specificare intenzionalmente percorsi errati e di utilizzare le informazioni restituite dall'eccezione per determinare i percorsi effettivi.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AddIn.Hosting.InvalidPipelineStoreException">
          <source>This exception is thrown by the following discovery methods that build and update the store of add-in and pipeline information: <ph id="ph1">&lt;xref:System.AddIn.Hosting.AddInStore.FindAddIns%2A&gt;</ph>,<ph id="ph2">&lt;xref:System.AddIn.Hosting.AddInStore.Rebuild%2A&gt;</ph>, <ph id="ph3">&lt;xref:System.AddIn.Hosting.AddInStore.RebuildAddIns%2A&gt;</ph>, <ph id="ph4">&lt;xref:System.AddIn.Hosting.AddInStore.Update%2A&gt;</ph>, and <ph id="ph5">&lt;xref:System.AddIn.Hosting.AddInStore.UpdateAddIns%2A&gt;</ph>.</source>
          <target state="translated">Questa eccezione viene generata dai metodi di individuazione seguenti che compilano e aggiornano l'archivio di componenti aggiuntivi e sulla pipeline: <ph id="ph1">&lt;xref:System.AddIn.Hosting.AddInStore.FindAddIns%2A&gt;</ph>,<ph id="ph2">&lt;xref:System.AddIn.Hosting.AddInStore.Rebuild%2A&gt;</ph>, <ph id="ph3">&lt;xref:System.AddIn.Hosting.AddInStore.RebuildAddIns%2A&gt;</ph>, <ph id="ph4">&lt;xref:System.AddIn.Hosting.AddInStore.Update%2A&gt;</ph>, e <ph id="ph5">&lt;xref:System.AddIn.Hosting.AddInStore.UpdateAddIns%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="T:System.AddIn.Hosting.InvalidPipelineStoreException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AddIn.Hosting.InvalidPipelineStoreException" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.AddIn.Hosting.InvalidPipelineStoreException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AddIn.Hosting.InvalidPipelineStoreException" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.AddIn.Hosting.InvalidPipelineStoreException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String)">
          <source>A description of the error.</source>
          <target state="translated">Descrizione dell'errore.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AddIn.Hosting.InvalidPipelineStoreException" /&gt;</ph> class with the specified message.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.AddIn.Hosting.InvalidPipelineStoreException" /&gt;</ph> con il messaggio specificato.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The serialized object data.</source>
          <target state="translated">Dati serializzati dell'oggetto.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The contextual information about the source or destination object data.</source>
          <target state="translated">Informazioni contestuali relative ai dati dell'oggetto di origine o di destinazione.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AddIn.Hosting.InvalidPipelineStoreException" /&gt;</ph> class with serialization and streaming context information.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.AddIn.Hosting.InvalidPipelineStoreException" /&gt;</ph> con le informazioni sulla serializzazione e sul contesto di flusso.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstitute an exception object that is transmitted over a stream.</source>
          <target state="translated">Questo costruttore viene chiamato durante la deserializzazione per ricostituire l'oggetto eccezione trasmesso in un flusso.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</source>
          <target state="translated">Per ulteriori informazioni, vedere <bpt id="p1">[</bpt>serializzazione XML e SOAP<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>A description of the error.</source>
          <target state="translated">Descrizione dell'errore.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Eccezione che ha determinato l'eccezione corrente.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not a null reference, the current exception is raised in a catch block that handles the inner exception.</source>
          <target state="translated">Se il parametro <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> non è un riferimento Null, l'eccezione corrente viene generata in un blocco catch che gestisce l'eccezione interna.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AddIn.Hosting.InvalidPipelineStoreException" /&gt;</ph> class with the specified message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.AddIn.Hosting.InvalidPipelineStoreException" /&gt;</ph> con il messaggio specificato e un riferimento all'eccezione interna che è la causa dell'eccezione.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Un'eccezione generata come risultato diretto di un'eccezione precedente deve includere un riferimento all'eccezione precedente nella proprietà <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed to the constructor, or a null reference if the <ph id="ph2">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">Il <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> proprietà restituisce lo stesso valore passato al costruttore o un riferimento null se il <ph id="ph2">&lt;xref:System.Exception.InnerException%2A&gt;</ph> proprietà non fornisce il valore di eccezione interna al costruttore.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of the <ph id="ph1">&lt;xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException&gt;</ph> class.</source>
          <target state="translated">Nella tabella riportata di seguito sono illustrati i valori iniziali delle proprietà per un'istanza della classe <ph id="ph1">&lt;xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Proprietà</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">Valore</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>The inner exception reference.</source>
          <target state="translated">Riferimento all'eccezione interna.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AddIn.Hosting.InvalidPipelineStoreException.#ctor(System.String,System.Exception)">
          <source>The localized error message string.</source>
          <target state="translated">Stringa del messaggio di errore localizzato.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>